src/selectors/breakpointSources.js

```diff
@@ -84,12 +83,17 @@ function findBreakpointSources(
   return sortBy(breakpointSources, (source: Source) => getFilename(source));
 }
 
-export const getBreakpointSources = createSelector(
+export const getBreakpointSources: Selector<
+  Array<{
+    source: Source,
+    breakpoints: FormattedBreakpoint[]
+  }>
```

Comment:
Maybe we pull this into its own type 

```js
type SourceBreakpoints = {|
   +source: Source,
   +breakpoints: FormattedBreakpoint[]
 |}
```

Reply:
You already had! I just missed it since it wasn't declared with the function.